docs/releasenotes.rst

```diff
@@ -1,23 +1,59 @@
 Release Notes
 =============
 
-0.10.0 (In Development)
------------------------
+1.0.0 (2016-01-04)
+----------------------------
 
-* Compatibility with Django 1.9 with Python 2.7 or Python 3.4/3.5
-* Compatibility with Django 1.8 with Python 2.7 or Python 3.4/3.5
+Release 1.0.0 is a comprehensive update of the django-timepiece codebase. It is
+based on 0.9.5, which was not officially released to PyPI. Release
+1.0.0 includes the bulk of the unofficial 0.9.5 release, along with several critical
+code quality improvements to 0.9.5.
+
+Please note that **release 1.0.0 is not backwards compatible with any 0.9.x release.**
+It is a new and separate refactoring of django-timepiece.
+
+*Features*
+
+* Assigning Project hours to a User not associated with the Project will automatically
+associate User with Project
+* Admins are now able to add Entries after User's timesheet has been approved
+* Payroll summary now groups by User last name
+* Dashboard comments have been converted to modal popups to keep interface clean
+* Contract detail now shows total hours worked before Contract start date and post
+Contract end date
+* Sticky headers have been added to all tables
+* Time Entries are now more precise to prevent the number of seconds being off in totals
+* Tabs for pending and completed Projects in the Contracts page
+* Users can only log activities that are allowed for a Project
+* Estimation Accuracy report has been refined for readability
+
+*Bugfixes*
+
+* Alphabetized entries in weekly schedule
+* Previous/Next options returned to the bottom of pages
+* Hourly Report outputs correct data
+* Sum of parts now accurately matches title (rounding display has been removed to show explicit
+times)
+* Dashboard header styling has been made consistent (hover styling removed and background color
+matched)
+* lessc installs locally rather than globally
+
+*Code Quality*
+
+* Compatibility with Django 1.6 with Python 2.7 or Python 3.3
+* Compatibility with Django 1.7 with Python 2.7 or Python 3.4
```

Comment:
This needs to be updated for 1.8 and 1.9 support on 2.7. or 3.5; the tox settings are a good metric of what we want to support.